web/src/click_handlers.js

```diff
@@ -592,7 +592,7 @@ export function initialize() {
         function check_reference_removed(mutation, instance) {
             return Array.prototype.includes.call(
                 mutation.removedNodes,
-                $(instance.reference).parents(".pm-list")[0],
+                $(instance.reference).parents(".dm-list")[0],
```

Comment:
I think a better pattern for commit messages is "left_sidebar: Rename pm-list or dm-list." or "left_sidebar: Rename pm-list to not mention pms." or "left_sidebar: Give pm-list a better name.".  "Use dm-list class." sounds like we're using an existing class where we were not using it previously.

Reply:
The next commit, "left_sidebar: Use DM identifiers around direct-messages list.", works well on this point.